src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleLines = sourceString.split(';');
+
+  return styleLines.reduce((stylesObject, line) => {
+    const trimmedLine = line.trim();
+
+    if (!trimmedLine) {
+      return stylesObject;
+    }
+
+    const [property, value] = trimmedLine.split(':').map((part) => part.trim());
+
+    if (property && value) {
+      stylesObject[property] = value;
+    }
+
+    return stylesObject;
+  }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Ensure that the chained `map` method inside the `split` has the same indentation level as the `split` method for consistency.

Reply:
After running the `npm t` command, the code is automatically converted into a single line.